app/build.gradle

```diff
@@ -66,7 +66,7 @@ dependencies {
     implementation project(':client')
     implementation 'androidx.appcompat:appcompat:1.5.1'
     implementation 'androidx.core:core-splashscreen:1.0.0'
-    implementation 'com.google.android.material:material:1.4.0'
+    implementation 'com.google.android.material:material:1.7.0'
```

Comment:
> I saw that the message view in the MessagesActivity is configured so that you can select the TextView content.
Unfortunately that conflicts with the onLongClickListener of the message item. I could not achieve to select any text. The onLongClickListener works.
When you decide to remove the isSelectable tag, then the longClickListener for the message item would also work when you touch the TextView and not only if you click outside of the TextView.

I'm okay if the copy complete message only works when long clicking outside the message content (image, title). But I think it's essential that you can select parts of the message. This only works in the PR when the contentType=text/markdown is set in the message, otherwise the text isn't selectable anymore. It did work previously.
[Peek 2023-02-03 18-13.webm](https://user-images.githubusercontent.com/14895212/216665409-2190d6cc-3325-4ab8-a4bc-7e0fcc3eb53f.webm)

I think just using the old TextView would fix the problem.

Reply:
Okay, let's see what I can do there.
Reverting one TextView is not that easy as any text view, even if implemented as 'TextView', is converted to 'MaterialTexiView' automatically.

Reply:
This problems seems to be well known for years now: https://issuetracker.google.com/issues/37095917  
But the mentioned workaround worked for me.

Please check if it does for you.

Reply:
It does :+1: